CONTRIBUTING.rst

```diff
@@ -51,3 +51,61 @@ All interactions for this project are covered by the
 `PSF Code of Conduct <https://www.python.org/psf/codeofconduct/>`_. Everyone is
 expected to be open, considerate, and respectful of others no matter their
 position within the project.
+
+
+Run pre-commit linting locally
+------------------------------
+
+If you'd like, you can run the project's basic linting suite locally,
```

Comment:
"the project's" feels odd verbiage for the PEPs repo -- perhaps out of pomposity, though

Reply:
I changed it to "this repo's"—is that better?